backend/diagram/class/ModelPackage.puml

```diff
@@ -78,7 +179,33 @@ package model {
         hide Status methods
     }
 
-   /'
+    package decorator {
```

Comment:
Hmm I think we'll have to delete this package. Its not being used. I was thinking of deleting it while refactoring. We may have to sync with @razine-bensari .